### PR TITLE
added extended collocation to metrics

### PR DESCRIPTION
**Pull request by alegrub88**

### CODE REVIEW

File: pytesmo/metrics.py [hunk 1]
```diff
@@ -342,6 +342,159 @@ def tcol_snr(x, y, z, ref_ind=0):
     return snr, np.sqrt(err_var) * beta, beta
 
 
+def EC(data, correlated=None, err_cov=None, abs_est=True):
```
Comment (cpaulik) [inline pytesmo/metrics.py]: Could you please make this lowercase `ec` or maybe `ecol` to be consistent with `tcol` in the other functions.

**Comment by cpaulik** [comment]
Thanks. I'll merge this after the function rename. The failed tests have nothing to do with this PR.

**Comment by alegrub88** [comment]
done.

**Comment by cpaulik** [comment]
Thanks. I'll release a new version when I have time to check why the tests are failing.

**Comment by sebhahn** [comment]
One or the other example how to use the extended collocation technique would be nice @alegrub88. Can you add some simple examples to the documentation?

**Comment by alegrub88** [comment]
I knew this was coming :-) 
examples are added and some missing imports/functions are added now.

**Comment by sebhahn** [comment]
see PR https://github.com/TUW-GEO/pytesmo/pull/127

**Pull request merged**